src/utils/shared-props.js

```diff
@@ -71,6 +71,7 @@ const PickerPropTypes = {
       imageUrl: PropTypes.string.isRequired,
     }),
   ),
+  notFound: PropTypes.object.isRequired,
```

Comment:
`notFound` should not be required here. You should be able to create the picker without adding a `notFound` prop.

Reply:
Also, can you please add documentation for this property to `README.md`?